packages/cmrjs/tests/test-cmr-utils.js

```diff
@@ -1,9 +1,14 @@
 const test = require('ava');
+const rewire = require('rewire');
 
 const {
   getGranuleId
 } = require('../cmr-utils');
 
+const cmrUtil = rewire('../cmr-utils');
+const isCMRFile = cmrUtil.__get__('isCMRFile');
```

Comment:
This stack overflow magic is my new jam. (it lets you test unexported functions without exporting them).